stardoc/stardoc.bzl

```diff
@@ -109,6 +109,9 @@ def _stardoc_impl(ctx):
                                 (ctx.label.name)),
         )
 
+    outputs = [out_file]
+    return [DefaultInfo(files = depset(outputs), runfiles = ctx.runfiles(files = outputs))]
```

Comment:
Please add comment mentioning this is a workaround for bazelbuild/bazel#15043 and to see discussion in bazelbuild/stardoc#139.

Reply:
Done